src/components/correct-name/CorrectNameRequest.vue

```diff
@@ -26,6 +26,7 @@
         <v-text-field
           id="nr-number"
           v-model="nrNumber"
+          autofocus
```

Comment:
We need to be careful when using this (because too many doesn't work well), but in specific cases this brings focus to the text field, which is very nice.

Reply:
Good to know!